integrated_channels/degreed2/admin/__init__.py

```diff
@@ -85,6 +85,7 @@ class Degreed2LearnerDataTransmissionAuditAdmin(admin.ModelAdmin):
         "enterprise_course_enrollment_id",
         "course_id",
         "status",
+        "modified",
```

Comment:
useful to see when the last attempt happened

Reply:
I've mentioned this a few times on similar PR's but would this field be useful on all customer config forms?

Reply:
yeah. will think about this...